public/Day 147/index.html

```
@@ -0,0 +1,1117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

HTMLHint (1.9.0)

[warning] The empty tag : [ meta ] must be self closed.

(tag-self-close)